recipe/meta.yaml

```diff
@@ -18,6 +18,7 @@ requirements:
   host:
     - python >=3.7
     - pip
+    - setuptools
```

Comment:
I am guessing this should also have `setuptools_scm` here?

cc: @chrisburr

Reply:
Good catch Saransh! 🙏

It appears that [the package uses `setuptools_scm[toml]` to build]( https://github.com/scikit-hep/mplhep_data/blob/242999e6704aa530f4b709b2ba05e1d5ef1fdfc7/pyproject.toml#L2 )

For context Chris, we are seeing the following error [in CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1069495&view=logs&jobId=c370752b-6889-5c3c-6797-af1cf6928e3e&j=c370752b-6889-5c3c-6797-af1cf6928e3e&t=ae5177ee-0cd0-5134-8c9e-7718a3efb233 ), which suggests the version metadata is not being captured in the package properly

```
+ pip check
mplhep 0.3.54 has requirement mplhep-data>=0.0.4, but you have mplhep-data 0.0.0.
```

@Saransh-cpp could you please open a PR to this feedstock that adds `setuptools_scm` and bumps the `build/number`?

Note: `setuptools_scm`'s [extra `toml` is empty]( https://github.com/pypa/setuptools-scm/blob/3c145dfd784e41cdb774449a421fe5e17a664786/pyproject.toml#L67-L68 ). This was converted to [a required dependency]( https://github.com/pypa/setuptools-scm/blob/3c145dfd784e41cdb774449a421fe5e17a664786/pyproject.toml#L45 ), which [conda-forge follows]( https://github.com/conda-forge/setuptools_scm-feedstock/blob/56e1bedbc6f504e65c9e1be5a7e4d455620285f4/recipe/meta.yaml#L31 ). So adding `setuptools_scm` should be sufficient